### PR TITLE
MNT Use GitHub Actions CI

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/gha-ci/issues/11<br /><br />Workflow run https://github.com/creative-commoners/silverstripe-graphql/actions?query=branch%3Apulls%2F3.8%2Fmodule-standards

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+          endtoend_suite: asset-admin
+          endtoend_config: vendor/silverstripe/asset-admin/behat.yml
+        - php: 7.4
+          endtoend: true
+          endtoend_suite: versioned-admin
+          endtoend_config: vendor/silverstripe/versioned-admin/behat.yml
+        - php: 7.4
+          endtoend: true
+          endtoend_suite: silverstripe-elemental
+          endtoend_config: vendor/dnadesign/silverstripe-elemental/behat.yml
```
Comment (ChloeSartorelli) [inline .github/workflows/ci.yml]: Why versioned and elemental? Those aren't mentioned in the travis config on this branch.
Also if we _should_ be including versioned-admin, should we do that for the 4.0 branch as well?

File: composer.json [hunk 2]
```diff
@@ -18,7 +18,8 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^9.5",
-        "squizlabs/php_codesniffer": "^3.0"
+        "squizlabs/php_codesniffer": "^3.0",
+        "silverstripe/frameworktest": "^0.4.3"
```
Comment (ChloeSartorelli) [inline composer.json:23]: Do we need to add versioned-admin and elemental as well?

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (emteknetnz) [reply .github/workflows/ci.yml]: I'm just copying what was previously in the travis file

Comment (emteknetnz) [reply .github/workflows/ci.yml]: oh, copied it from the wrong branch :-) 3.8 only has asset-admin. 4.0 has asset-admin, versioned-admin and elemental

File: composer.json (hunk 2 above)
Comment (emteknetnz) [reply composer.json:23]: no, only supposed to have asset-admin in this one, have added to require-dev

**Review by GuySartorelli — APPROVED** [review]

**Pull request merged**